src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_SummaryScreen/TicketAssistant_SummaryScreen.tsx

```diff
@@ -143,6 +142,16 @@ export const TicketAssistant_SummaryScreen = ({navigation}: SummaryProps) => {
             </ThemeText>
 
             <TicketSummary />
+            {doesTicketCoverEntirePeriod && (
+              <MessageBox
+                style={styles.infoBox}
+                type={'info'}
+                title={t(TicketAssistantTexts.summary.durationNotice.title)}
+                message={t(
+                  TicketAssistantTexts.summary.durationNotice.description,
+                )}
+              ></MessageBox>
```

Comment:
Doesn't need to have `</MessageBox>` when there is no children. Just end the tag with `/>`